### PR TITLE
Release v6.1 lacework-agent Helm Charts

### PR DESCRIPTION
**Pull request by c-p-i-o**
Release notes are at:
https://docs.lacework.com/releases/2022-12_october-agent-release

Signed-off-by: Chirag Pandya <chirag.pandya@lacework.net>

### CODE REVIEW

File: lacework-agent/values.yaml [hunk 1]
```diff
@@ -35,14 +35,15 @@ clusterAgent:
   image:
     registry: docker.io
     repository: lacework/k8scollector
-    tag: latest
+    tag: 6.1.0
```
Comment (c-p-i-o) [inline lacework-agent/values.yaml:38]: @rnalexlacework @cirego - shall I leave this tag as unaltered (i.e. leave it as `latest`?)

Comment (rnalexlacework) [reply lacework-agent/values.yaml:38]: My guess is we should not pin it to 6.1.0.  May be @cirego or @nschmeller have a better idea.

File: lacework-agent/templates/configmap.yaml [hunk 2]
```diff
@@ -54,3 +60,7 @@ data:
               {{- if .Values.laceworkConfig.kubernetesCluster }}, "KubernetesCluster" :"{{ .Values.laceworkConfig.kubernetesCluster}}" {{- end }}
              }
     }
+
+{{- with .Files.Glob "config/syscall_config*.yaml" }}
```
[the remark this answers is not in the document]
Comment (rnalexlacework) [reply lacework-agent/templates/configmap.yaml:64]: We can remove this. If the customer does mv syscall_config.yaml to syscall_configold.yaml which is probably not uncommon we might include both in the configmap.

**Review by rnalexlacework — APPROVED** [review]

**Review by anilnanduri — APPROVED** [review]

File: lacework-agent/values.yaml [hunk 3]
```diff
@@ -35,14 +35,15 @@ clusterAgent:
   image:
     registry: docker.io
     repository: lacework/k8scollector
-    tag: latest
+    tag: latest
```
Comment (cirego) [inline lacework-agent/values.yaml:38]: Nit: trailing space

**Review by cirego — APPROVED** [review]

**Pull request merged**